src/components/useSwipeable/index.ts

```diff
@@ -12,15 +12,34 @@ export const DEFAULT_CONFIGURATION = {
   offsetPerNode: 225,
 };
 
-export const useSwipeable = (config: SwipeableConfiguration = {}) => {
+export const useSwipeable = (numItems: number, config: SwipeableConfiguration = {}) => {
   const configuration = { ...DEFAULT_CONFIGURATION, ...config };
   const { swipeSensitivity = 25, deceleration = 0.8, offsetPerNode } = configuration;
-  const [offset, setOffset] = useState<number>(0);
+  const [offset, _setOffset] = useState<number>(0);
   const [isMoveInProgress, setIsMoveInProgress] = useState<boolean>(false);
   const [currentX, setCurrentX] = useState<number>(0);
   const [prevMoveTime, setPrevMoveTime] = useState<number>(0);
   const [velocity, setVelocity] = useState<number>(0);
 
+  const setOffset = useCallback(
```

Comment:
the variable naming could be improved here like instead of `_setOffset` something like `setOffsetForced` or naming this one `setOffsetWithSnap`. the convention at premier software development firm dovetail is underscore prefix is reserved for unused variables that you can't avoid defining.

Reply:
can't argue with dubtail conventions, will do!